cornell-dti/course-plan
# Team feedback major fixes

### Summary 

This pull request addresses the most major issues found from team feedback.

- [x] Fix editing semester bugs (editing the wrong semester or not editing at all)
- [x] Replace the new semester button with one at the top of the semester view (aligned with the first semester if there are semesters on both views, otherwise aligned on the left)
- [x] Update A&S credit requirement

### Test Plan 

- Verify that editing semesters works in all situations (especially when editing the first semester after creating 2 new ones) and on both compact & regular.
- Ensure semesters can still be added with the new button per usual on compact, regular, and mobile views. Also ensure it can be used in all of these states when you have no semesters in your plan. 
- Ensure the A&S 100 credit requirement can be applied to all classes that end in a distribution -AS (like some CS/INFO courses)
